lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -6012,12 +6015,12 @@ struct AtomicRMWOpConversion
         return failure();
       }
       atom.o(rmwOp).o(sTy);
-
-      atom(dstOpr, ptrOpr, valOpr);
-      auto ret = ptxBuilder.launch(rewriter, loc, valueElemTy, false);
+      Value pred = int_val(1, 1);//TODO:[dongdongl] actual mask 
```

Comment:
Normally comments shouldn't appear in the same line with the code for coding style. Something like this should be better:

//TODO:[dongdongl] actual mask
Value pred = int_val(1, 1);

Reply:
Done